src/Microsoft.ML.Data/Evaluators/Metrics/MulticlassClassificationMetrics.cs

```diff
@@ -114,9 +120,10 @@ internal MulticlassClassificationMetrics(IHost host, DataViewRow overallResult,
             MacroAccuracy = FetchDouble(MulticlassClassificationEvaluator.AccuracyMacro);
             LogLoss = FetchDouble(MulticlassClassificationEvaluator.LogLoss);
             LogLossReduction = FetchDouble(MulticlassClassificationEvaluator.LogLossReduction);
+            TopKAccuracyForAllK = RowCursorUtils.Fetch<VBuffer<double>>(host, overallResult, MulticlassClassificationEvaluator.AllTopKAccuracy).DenseValues().ToImmutableArray();
```

Comment:
(pinned to unrelated code to allow for threaded discussion)

So I believe that the `GetOverallMetricColumns()` also needs to be updated to include the new metric:

https://github.com/dotnet/machinelearning/blob/a9ab7fc530ba9a2a519aaca70862ee67f42bf712/src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs#L127-L137

It seems that the `GetOverallMetricColumns()` method is used only by MAML (which is a deprecated tool that was used to be created on top of ML.NET) and the Sweeper project (which I also believe isn't publicly available through ML.NET's API), and even then I'm not sure what was the purpose of the method. So I'm not sure if we should worry much about this.

But in general I'd only add the following `yield` statement to the code, just in case:

```C#
yield return new MetricColumn("TopKAccuracyForAllK", TopKAccuracyForAllK, isVector: true);
```